tests/test_errata_data.py

```diff
@@ -226,10 +227,14 @@ def test_save_product_component_for_errata(
         )
     build_list_url = f"{settings.ERRATA_TOOL_URL}/api/v1/erratum/{erratum_id}/builds_list.json"
     requests_mock.get(build_list_url, text=build_list)
+    sb = SoftwareBuildFactory(build_id="1636922", build_type=BUILD_TYPE)
     slow_load_errata(erratum_id)
-    pcr = ProductComponentRelation.objects.filter(external_system_id=erratum_id)
-    assert len(pcr) == no_of_objs
+    pcrs = ProductComponentRelation.objects.filter(external_system_id=erratum_id)
+    assert len(pcrs) == no_of_objs
     assert mock_send.call_count == no_of_objs
+    for pcr in pcrs:
+        if pcr.build_id == "1636922":
+            assert pcr.software_build_id == sb.pk
```

Comment:
Maybe below is a little cleaner:
```suggestion
        # If the relation uses this build's ID
        if pcr.build_id == sb.build_id:
            # assert it is linked to the build using the ForeignKey field
            assert pcr.software_build_id == sb.pk
        else:
            # else assert the ForeignKey is unset / other build IDs have not been fetched
            assert pcr.software_build_id is None
```